runbooks/key-handover/SignFromJson.s.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+
+contract SignFromJson is OriginalSignFromJson {
```

Comment:
I'm not sure we want to be duplicating code in the runbooks directory. 

This file is already included as part of https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/mmz-002-key-handover